.github/workflows/linux_riscv_xuantie_dev_cpu.yml

```diff
@@ -2,13 +2,21 @@ name: Linux RISC-V with Xuantie (Ubuntu 22.04, Python 3.10)
 on:
   workflow_dispatch:
     inputs:
-      testFilter:
-        description: 'Filter for google tests'
+      testFilterType:
+        description: 'Filter for google tests {SMOKE | CONCISE}'
         required: true
-        default: '*smoke_AdaPoolAvg4DLayoutTest*'
+        default: 'CONCISE'
   pull_request:
     paths:
       - '.github/workflows/linux_riscv_xuantie_dev_cpu.yml'
+      - 'src/plugins/intel_cpu/src/emitters/plugin/aarch64/**'
+      - 'src/plugins/intel_cpu/src/emitters/plugin/riscv64/**'
+      - 'src/plugins/intel_cpu/src/emitters/plugin/x64/**'
+      - 'src/plugins/intel_cpu/src/nodes/executors/aarch64/**'
+      - 'src/plugins/intel_cpu/src/nodes/executors/shl/**'
+      - 'src/plugins/intel_cpu/src/nodes/kernels/riscv64/**'
+      - 'src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/riscv64/**'
+      - 'src/plugins/intel_cpu/thirdparty/xbyak_riscv/**'
```

Comment:
By the way, I noticed that there are extra paths for triggering (which are not related to RISC-V) which should be removed:
```suggestion
      - '.github/workflows/linux_riscv_xuantie_dev_cpu.yml'
      - 'src/plugins/intel_cpu/src/emitters/plugin/riscv64/**'
      - 'src/plugins/intel_cpu/src/nodes/executors/shl/**'
      - 'src/plugins/intel_cpu/src/nodes/kernels/riscv64/**'
      - 'src/plugins/intel_cpu/src/nodes/eltwise.**'
      - 'src/plugins/intel_cpu/src/nodes/fullyconnected.**'
      - 'src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/riscv64/**'
      - 'src/plugins/intel_cpu/thirdparty/xbyak_riscv/**'
```
*Added two new paths instead